mycore-restapi/src/main/java/org/mycore/restapi/v1/utils/MCRRestAPIObjectsHelper.java

```diff
@@ -143,14 +154,40 @@ public static Response showMCRObject(String pathParamId, String queryParamStyle,
             }
         }
 
+        String xslfile = MCRConfiguration.instance().getString("MCR.RestAPI.v1.Filter.XML", "");
```

Comment:
Please use [MCRContentTransformerFactory](http://www.mycore.de/generated/mycore-2017.06.0/mycore-base/apidocs/org/mycore/common/content/transformer/MCRContentTransformerFactory.html) to do transformations for better performance and more flexibility